docker-compose.yml

```diff
@@ -7,15 +7,17 @@ x-services: &env
 services:
   wamp:
     <<: *env
-    build: ./wamp
+    build: wamp
     environment:
       - NEAR_EXPLORER_WAMP_BACKEND_SECRET=THIS_IS_LOCALHOST_SECRET
     ports:
       - 10000:8080
 
   backend:
     <<: *env
-    build: ./backend
+    build:
+      context: .
+      dockerfile: backend/Dockerfile
```

Comment:
Just for the future reference, I feel it is an ugly hack that is only used to allow cross-backend-frontend imports (which is a smelly code that we all know). We should address it one day, yet in order to unblock this major refactoring, I will approve this PR since it is in a working state.